README.md

```diff
@@ -0,0 +1,386 @@
+# 📖 RubyOnRails-Blog<a name="about-project"></a>
+> The RubyOnRails-Blog app will be a classic example of a blog website. I will create a fully functional website that will display a list of posts and empower readers to interact with them by adding comments and liking posts.
+
+<!-- > Describe your project in 1 or 2 sentences. -->
+### How to build the RubyOnRails-Blog app
+
+The end result should follow the following data model (this is an Entity Relationship Diagram):
+
+<p align="center">
+  <img src="./public/blog_app_erd_v1_1.png" alt="Data model"  width="500px"  />
+</p>
+
+For this project I will have full freedom in terms of visual design but I will need to keep the following wireframes:
+
+<p align="center">
+  <img src="./public/blog_login.png" alt="Login page wireframe" width="350px" />
+  <img src="./public/blog_root_users.png" alt="All users page wireframe"  width="350px"  />
+  <img src="./public/blog_user_page.png" alt="Single user page wireframe"  width="350px"  />
+</p>
+
+<p align="center">
+  <img src="./public/blog_user_all_posts.png" alt="All posts by a given user page wireframe"  width="350px"  />
+  <img src="./public/blog_single_post.png" alt="Single post page wireframe"  width="350px"  />
+</p>
+
+I will practice my knowledge about basic Ruby on Rails concepts - starting from the Views and finishing the database connection. I will also implement ideas like authentication, authorization, and all kinds of tests. In the last step, I will create an API instead of API consumption.
+
+### Projects list
+I will be building the RubyOnRails-Blog for three weeks. Here is the list of projects that will guide me through the steps described above. Please find details about each of the milestone requirements in the upcoming project activities.
+
+- Project 1: Creating a data model.
+  - Create a new rails app. (rails new app-name --database=postgresql)
+  - add database credentials at config\database.yml
+  - run `rails db:create`
+  - Build project schema.
+    - Create the necessary migration files.
+      - `rails generate migration CreateUsers name:string photo:string bio:text posts_counter:integer`
+      - `rails generate migration CreatePosts title:string text:text comments_counter:integer likes_counter:integer`
+      - `rails generate migration CreateComments text:text`
+      - `rails generate migration CreateLikes`
+    - Run the necessary migration files.
+      - `rails db:migrate`
+    - Table and column names should match the ERD diagram.
+      - note: photo for users table should be a link to an image
+    - Foreign keys should be included.
+      - Pay attention that in `posts` table the column holding the foreign key to the `users` table should be named `author_id`
+    - All columns that are foreign keys should have a corresponding index.
+  - Be sure to reference the from the ERD.
+    <p align="center">
+      <img src="./public/blog_app_erd_v1_1.png" alt="Data model" width="500px" />
+    </p>
+
```

Comment:
- [ ] There are several broken links in your README file which make it look unprofessional. Kindly make sure that you push the image that you're referring to your repo or simply remove the links from your README file.